src/main/webapp/templates/userAssignedDialog.hbs

```diff
@@ -1 +1,2 @@
-<h1>Issue {{issueKey}} assigned to {{prettyName}}</h1>
\ No newline at end of file
+<select:user id="user"/><br/>
```

Comment:
Are you assuming the select primitive will show the selected user?

Reply:
The current behaviour do that

Reply:
I think we should remove this component for while. Once the bug is fixed we can put this again.